Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSSave.py

```diff
@@ -134,6 +137,16 @@ def PyExec(self):
         file_name = self.getProperty("Filename").value
         workspace = self.getProperty("InputWorkspace").value
 
+        sample = workspace.sample()
```

Comment:
While I was testing I realised that you can change the thickness, shape, height and width values in the batch table of the SANS interface. I'm afraid I'm not familiar enough with the interface or code to know how this is intended to be used, but I noticed that because we're taking these values from the workspace when we're saving it means that any changes to the batch table are not saved out. Is this right, or should we be saving whatever is in the batch table?